### PR TITLE
use any-observable

### LINKED ISSUE #5: Refactor to use `any-observable`
I am thinking of dropping the `create` method entirely.

If people want to use specific extended Obvservable methods of `rxjs` or other implementations, they should use `any-observable/register` to do it.

### PR DESCRIPTION
**Pull request by jamestalmage**
Fixes #5

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,5 +36,8 @@
     "rxjs": "^5.0.0-beta.6",
     "xo": "^0.14.0",
     "zen-observable": "^0.2.1"
+  },
+  "dependencies": {
```
Comment (sindresorhus) [inline package.json]: I would put this above devDependencies for consistency.

**Comment by sindresorhus** [comment]
LGTM

**Comment by jamestalmage** [comment]
I wish coveralls would default to not using those comments. Especially since it knows the status API is available

**Comment by sindresorhus** [comment]
@jamestalmage First thing I do when activating coveralls is disabling those annoying comments.

**Comment by jamestalmage** [comment]
Yeah - I usually do to. Forgot on this one Obviously

**Pull request merged**